tests/non_silent.rs

```diff
@@ -0,0 +1,48 @@
+// pest. Elegant, efficient grammars
+// Copyright (C) 2016  Dragoș Tiselice
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#[macro_use]
+extern crate pest;
+
+use pest::prelude::*;
+
+impl_rdp! {
+    grammar! {
+        abc = { ["a"] ~ ["b"] | ["a c"] }
+
+        whitespace = { [" "] }
```

Comment:
I meant inside a rule, let's say i want to keep whitespace in general but not between a given rule, something like `if_tag = { tag_start ~ ["if"] ~ expression ~ tag_end }` where i do not care about whitespace in between but it can exist.